packages/headless/src/controllers/standalone-search-box/headless-standalone-search-box.ts

```diff
@@ -144,7 +143,10 @@ export function buildStandaloneSearchBox(
       return {
         ...searchBox.state,
         isLoading: standaloneSearchBoxState.isLoading,
-        redirectTo: standaloneSearchBoxState.redirectTo,
+        redirectTo:
```

Comment:
The type is string | null, but in the slice's state, it's only a string. This poses a bit of a comprehension problem.

Reply:
We could adjust the type to only be a `string` here too? Starting to send `null` could be breaking if someone is checking for an empty string.

e.g.

```
if (redirectTo === '') {
  // do nothing
}
```

Reply:
Im afraid that's breaking to change the type as well, or misleading? Before the new slice, I think we actually returned null?

Reply:
We would also need to change the usage doc to check of an empty string

Reply:
The old slice was indeed sending both values, whereas the new one just returns `string`. The null type is removed inside `fetchRedirectUrl`.

Constraining a return type is not breaking because a caller is expected to be handling all return types. Ofc, if someone is handling `null` and not empty string, there would be an impact, but I think most are doing:

```
if (redirectionUrl) {
  // do nothing
}
```

Some reasons to remove `null`:
- The less return types to check, the easier it is for a consumer.
- If someone configures the `redirectionUrl` option an empty string, it feels more correct to return an empty string rather than `null`.
- One less ternary.

> We would also need to change the usage doc to check of an empty string

Which part of the doc would need to change?